js/scripts/util.js

```diff
@@ -176,12 +176,21 @@ const checkBlockCollisions = (entity) => {
 /** Contains all functions called as event handlers. */
 const EVENT_HANDLERS = {
     gameplayMouseDown: (e) => {
-        GAME.user.firing = false;
-        GAME.user.aiming = true;
+        // check if mouse button is left click
+        if (e.button === 2) {
```

Comment:
Could we update some of the naming here? e.button === 2 is confusing. Perhaps e.click === RIGHT_CLICK or something like that?